pyensembl/ensembl_release.py

```diff
@@ -60,8 +60,8 @@ def _delete_cached_files(self):
         is assumed to be some view of the this release's data and thus
         safe to delete.
         """
-        base = self.gtf.base_filename()
-        dirpath = self.local_gtf_dir()
+        base = self.gtf.local_filename()
```

Comment:
I think `base_filename` is actually what you want here, since all the locally cached files look like `base_filename + "_something_extra.csv"`